lib/diameter/Makefile

```diff
@@ -28,5 +28,6 @@ SPECIAL_TARGETS =
 include $(ERL_TOP)/make/otp_subdir.mk
 
 DIA_PLT_APPS=ssl runtime_tools syntax_tools
+TEST_NEEDS_RELEASE=true
```

Comment:
One of the nice things about using `ct_run` to run tests instead of `ts` is that we get all the features of ct without having to update ts to support them. So, if anything I would like to see us converge on using `ct_run` instead of `ts` if possible.